<launch>
  <node name="execute" pkg="formation_control" type="execute" output="screen"/>
  <node name="srv_call" pkg="formation_control" type="srv_call" output="screen"/>
</launch>